docs/2-building-blocks-of-oop-part-1/hometask/src/Point.ts

```diff
@@ -0,0 +1,36 @@
+export class Point {
+  private x: number;
+  private y: number;
```

Comment:
should not be `private`

For the current task `readonly` modifiers will be the better choice.